L1Trigger/L1TTrackMatch/interface/L1TkHTMissEmulatorProducer.h

```diff
@@ -26,10 +26,10 @@ namespace l1tmhtemu {
 
   // extra room for sumPx, sumPy
   const unsigned int kEtExtra{10};
-
   const unsigned int kValidSize{1};
-  const unsigned int kMHTSize{15};     // For output Magnitude default 15
-  const unsigned int kMHTPhiSize{14};  // For output Phi default 14
+  const unsigned int kMHTSize{16};  // For output Magnitude default 15
+  const unsigned int kMHTIntSize{11};
+  const unsigned int kMHTPhiSize{13};  // For output Phi default 14
```

Comment:
Still valid this `default 14`?

Reply:
@Chriisbrown can you answer this question?